### DISCUSSION TITLE
isSelected from original doesn't have effect on the row.

### DISCUSSION THREAD

**Discussion by Earlo** [TanStack/table#3080] [Q&A]
I'm trying to have a row selected by default. However, when I pass data with a isSelected defined, it doesn't have effect on the row `useTable` gives me.

![image](https://user-images.githubusercontent.com/12685733/108339666-5eade600-71e0-11eb-9afe-e9a09df239fa.png)

**Comment by N1ck0lay** [comment]
Can some one help with this?
